client/src/index.js

```diff
@@ -1,17 +1,16 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { ConfigProvider } from 'antd';
+import frFR from 'antd/lib/locale/ar_EG';
+
 import App from './App';
-import reportWebVitals from './reportWebVitals';
 
 ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
+  <Router>
+    <ConfigProvider locale={frFR}>
```

Comment:
Our website is an RTL Arabic website not french, so we should pass the right direction and language, and even if you imported the right language `'antd/lib/locale/ar_EG'` you should declare it with the appropriate name such as `Arabic`

```suggestion
    <ConfigProvider direction="rtl" locale={Arabic}>
```